lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextLifeCycle.cs

```diff
@@ -48,7 +48,7 @@ internal sealed class ContextLifeCycle
             _contextMessageHandlers = contextMessageHandlers;
         }
 
-        public string Id { get; private set; }
+        public string Id { get; }
```

Comment:
Yes. I used ReSharper to automate the change and checked the result, all these `set` are not used and not publicly visible (so don't need to deprecate). And from the code it looks like the properties should be read-only indeed and not modified during runtime